lib/kitchen/provisioner/chef_zero.rb

```diff
@@ -76,14 +76,18 @@ def run_command
 
         prefix_command(
           wrap_shell_code(
-            [cmd, *chef_client_args].join(" ").
+            [cmd, *chef_client_args, last_exit_code].join(" ").
```

Comment:
Yeah, unfortunately, to get the real exit code from PowerShell, we've got to do some gymnastics after chef-client exits.